### ISSUE TITLE
Enable producers to access their products on BOM and edit quantity / weight

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/wishlist#281]
## What is the need / problem?

Big hubs are spending too much time asking each producer for their final weight per products.

they want their producer to edit themselves.

We need to be careful about the setting "show customers names". Perhaps to make it easier: if producer are allowed to acces BOM, they are allowed to see customer names (the two radio button would be linked).

## Which type of users does this problem affect (and how many, if known)?

_- Hub Managers (MCFEs)_
_- Producers_

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]

Setting in shop preferences + permission for producer to see only their products in BOM, per shop they are selling to.

**Comment by tschumilas** [comment]
There is a pre-existing issue related to the same problem.  Agree its a significant problem.   Originally we (I think it was @kirstenalarsen involved in this discussion with me originally) thought giving producers permission to edit their products on orders would be a solution --   https://github.com/openfoodfoundation/wishlist/issues/172

**Comment by RachL** [comment]
@tschumilas yes I was hoping it would be easier to start on BOM page because BOM is listing line items and not orders (so no big design needed or weird cases with empty orders...). But not an easy fix...
